substrate/frame/salary/src/lib.rs

```diff
@@ -382,6 +394,54 @@ pub mod pallet {
 		}
 	}
 
+	#[pallet::tasks_experimental]
+	impl<T: Config<I>, I: 'static> Pallet<T, I> {
+		#[pallet::task_list(vec![].into_iter())]
```

Comment:
we could have an actual task list: return a vector with one element `()` if `now >= startus.cycle_start+cycle_period`, no?

Reply:
Thanks for review!

Debugging will be easier with this.

Reply:
I don't understand what you mean by "outputs in a block"

AFAIK the task list can return all the tasks available.
So I expect a code like this:

```
#[pallet::task_list({
	let now = frame_system::Pallet::<T>::block_number(); 
	let status = Status::<T, I>::get().expect("Task require active status");
	let cycle_period = Pallet::<T, I>::cycle_period();
	let result = if now >= status.cycle_start + cycle_period {
		vec![()] // Success one task available: `()`.
	} else {
		vec![] // Failure no task available.
	};
	result.into_iter()
})]
```

This is what I would expect looking at the example: https://paritytech.github.io/polkadot-sdk/master/frame_support/pallet_macros/attr.tasks_experimental.html#example